lib/Default/SmrPlanet.class.inc

```diff
@@ -847,6 +848,8 @@ class SmrPlanet {
 		$this->delayedShieldsDelta = 0;
 		$this->setCDs($this->getCDs(true));
 		$this->delayedCDsDelta = 0;
+		$this->setArmour($this->getArmour(true));
```

Comment:
I think this might be able to cause issues with when the armour hits 0, at the same time delayed armour is possible to be added, so it should be dealt with here and this just reduces the possible bugs (I don't think anywhere actually adds it though)